### ISSUE TITLE
Syntax error: un-terminated Hollerith code. Wrong detection of Hollerith code.

### ISSUE THREAD

**Issue by clementval** [omni-compiler/omni-compiler#192]
When we try to use a variable including digit and an `h/H` for a parameter, the front-end detect it as an un-terminated Hollerith code. This is wrong. 

```fortran
MODULE mod1
  IMPLICIT NONE

  INTEGER , PARAMETER :: &
   i_nol=0,     &
   i_mid=1,     &
   i_bnd=2,     &
   n_ppos=i_bnd,&
   i_low=0,     &
   i_sur=1,     &
   i_2m =2,     &
   i_10m=3,     &
   n_hpos=i_10m,&
   i_850hPa=4,  & 
   n_dpos=i_850hPa

END MODULE mod1
```

```bash
"mod1.f90:mod1", line 15: warning: un-terminated Hollerith code.
"mod1.f90:mod1", line 4: parameter value not specified
"mod1.f90:mod1", line 4: syntax error
```

**Comment by clementval** [comment]
The problem seems to be related with the `_` in the declaration of the parameters. If I remove them, everything is fine.

**Comment by clementval** [comment]
The example can be reduced to this simple one: 

```fortran 
MODULE mod1
  IMPLICIT NONE

  INTEGER , PARAMETER :: i_850hPa=4 

END MODULE mod1
```

**Comment by clementval** [comment]
@h-murai Pull Request #193 fixes this issues. Maybe this Hollerith code detection should only be enabled if we are using `-f77` or `-f90`. In `-f95`, there should be no Hollerith code.

**Comment by clementval** [comment]
Fixed by #193
